lib/udf_findhotel_utils.rb

```diff
@@ -163,10 +163,17 @@ def get_label_value(label, *keys):
           body:        %~
             from urlparse import urlparse, parse_qsl
             import json
-            return json.dumps(dict(parse_qsl(urlparse(url)[2])))
+            res = {}
+            try:
+              res = json.dumps(dict(parse_qsl(urlparse(url)[2])))
+            except:
```

Comment:
Let's catch a more narrow exception so it does not silence other problems that might happen in the future. `UnicodeDecodeError` seems to be the one, right ?